packages/components/src/Components/DownloadButton/DownloadButton.js

```diff
@@ -33,7 +33,7 @@ class DownloadButton extends Component {
                         iconComponent={null}
                         onToggle={ this.onToggle }
                     >
-                        <ExportIcon size={ IconSize.sm } />
+                        <ExportIcon size="sm" />
```

Comment:
So what's the convention, single quotes or double quotes? I see a lotta each of em used (or is it kinda 🤷‍♂ )

Reply:
Double quotes for html-like and single quotes for JS